Core/clim-core/frames.lisp

```diff
@@ -252,6 +252,9 @@ documentation produced by presentations.")
   ;; Let client code know.
   (clime:note-frame-pretty-name-changed (frame-manager frame) frame new-value))
 
+(defmethod frame-all-layouts ((frame application-frame))
+  (map 'list #'car (frame-layouts frame)))
```

Comment:
I use `map` when I'm treating the argument as a sequence, not specifically as a list (that is, in contrast to  `mapcan`, `mapl` and friends).

If we want to enforce a particular policy with these matter-of-taste-ish things (line breaking, helper functions and `&aux` come to mind), we should write a coding style document instead of discussing this in each pull request.

Reply:
I'm asking, because you've changed the call which was used in a macro expansion of the same method. I.e is there a sudden reason that frame-layouts returns a vector (not always a list)? if not, then using map removes this semantic information from this code (that frame-layouts returns specifically a list).

In the past we were discussing "matter of taste" things in pull requests number of times; proposing a coding style document makes sense of course, but I don't think it makes this remark "sudden" in this pull request.

Reply:
please use list operators where we deal with lists (as you have proclaimed in the first paragraph of the answer).

Reply:
Changed.